### PR TITLE
Infra drilldown

### PR DESCRIPTION
**Pull request by CharlieWinters**

### CODE REVIEW

File: src/modules/EventTypes.js [hunk 1]
```diff
@@ -11387,7 +11387,75 @@ export const EventTypes = {
...
+                name: 'apmApplicationNames'
+              },
+              {
+                label: 'NR APM Application Ids',
+                name: 'nr.apmApplicationIds'
+              },
+              {
+                label: 'NR APM Application Ids',
+                name: 'nr.apmApplicationNames'
+              },
```
Comment (jospdeleon) [inline src/modules/EventTypes.js:11442]: Do we want to expose the NR attributes here?

File: src/modules/EventTypes.js [hunk 2]
```diff
@@ -11387,7 +11387,75 @@ export const EventTypes = {
...
+                name: 'entityKey'
+              },
+              {
+                label: 'Instance Type',
+                name: 'instanceType'
+              },
+              {
+                label: 'NR Entity Type',
+                name: 'nr.entityType'
+              },
```
Comment (jospdeleon) [inline src/modules/EventTypes.js:11422]: Do we want to expose the NR attributes here?

Comment (CharlieWinters) [reply src/modules/EventTypes.js:11422]: These attributes come in on the 1st party and accounts got points for custom attributes where there were no custom attributes as these were included, so as this is shared between 3rd party and 1st party I've added them here. An alternative could be to override and augment this in 1st party.

**Comment by CharlieWinters** [comment]
Thanks @jospdeleon, I think I've implemented all the suggested changes, let me know in the ticket if there's anything else.

**Comment by jospdeleon** [comment]
Looks good. I fixed the es lint errors and merged them.

**Pull request merged**